src/components/drawing/Canvas.js

```diff
@@ -111,16 +111,16 @@ class Canvas extends Component {
         this.props.onRedoClick();
     }
 
-    renderHandles(shape) {
+    renderHandles(shape, scale) {
```

Comment:
Rather than passing scale into this function can we just add it to the line below?
`const { propagateEvents, matrix } = this.props;`

I can't really ever think of a reason why we would want to pass a different scale into this function besides the one defined in the props, so I think it makes more sense to not be in the arguments of the function

Reply:
done!

Reply:
done!